### ISSUE TITLE
Questions regarding 3-credit and 4-credit section

### ISSUE THREAD

**Issue by bojiang3** [uofi-stat430/fa20-disc#26]
Hi, is there any differences between the 3-credit undergraduate section and the 4-credit section grad section in this course except for additional homework problems? I have gone through the syllabus but didn't find any information indicating differences besides homework. Would you accept the request if some undergrads would like to take the 4-credit section?

**Comment by eddelbuettel** [comment]
Undergraduate students have always been allowed to take the graduate student homework questions as additional credit, subject to not being able to score more than 100%.

Put more simply, say undergrad question points total 50 and got 90% correct for 45 points.  By scoring five or more points correctly from the graduate student homework questions you can push your score from 45 back to 50.   

Hope this helps.  This policy has been consistent between the Sp19, Fa19 and Fa20 versions of this course.

**Comment by eddelbuettel** [comment]
> Would you accept the request if some undergrads would like to take the 4-credit section?

If you mean formally in order to obtain four credits as an undergraduate  -- you would have to check with the department office first to see if that actually works, which I somehow doubt.

**Comment by bojiang3** [comment]
Thank you for your reply. I meant formally obtain 4 credits as an undergrad since I am interested in a deeper insight into this course. I will contact the department shortly.

**Comment by eddelbuettel** [comment]
Yes, sorry. I misread your question wrong.   I suspect the answer is not as the college approved the course for your year and grade under a certain standard credit scheme, and I am not aware of one-offs that allow you to increase credit.  But hey I may well be unaware of things so by all means please inquire, and then report back here.

**Comment by bojiang3** [comment]
You are good. I just got the reply, and unlike the CS department approving some cases, the STAT department do not allow undergraduate students to take the graduate section

**Comment by bojiang3** [comment]
Thanks for your response
